src/core_functions/aggregate/nested/list.cpp

```diff
@@ -208,10 +191,13 @@ unique_ptr<FunctionData> ListBindFunction(ClientContext &context, AggregateFunct
 }
 
 AggregateFunction ListFun::GetFunction() {
-	return AggregateFunction({LogicalType::ANY}, LogicalTypeId::LIST, AggregateFunction::StateSize<ListAggState>,
-	                         AggregateFunction::StateInitialize<ListAggState, ListFunction>, ListUpdateFunction,
-	                         ListCombineFunction, ListFinalize, nullptr, ListBindFunction, nullptr, nullptr,
-	                         ListWindow);
+	auto func =
+	    AggregateFunction({LogicalType::ANY}, LogicalTypeId::LIST, AggregateFunction::StateSize<ListAggState>,
+	                      AggregateFunction::StateInitialize<ListAggState, ListFunction>, ListUpdateFunction,
+	                      ListCombineFunction, ListFinalize, nullptr, ListBindFunction, nullptr, nullptr, nullptr);
+	func.absorb = ListAbsorbFunction;
```

Comment:
Do we need a new `AggregateFunction` constructor that allows passing `absorb`, or is this something that we can consider if we have other aggregate functions (not just `LIST`) that use `absorb`? I assume we cannot change the existing constructors because we cannot change the API?

Reply:
We could but I wanted the default to automatically set `absorb` to `combine`. `LIST` is the only example know of, although I might make the sorted wrapper use it once it gets lists.